src/collection-browser.ts

```diff
@@ -382,6 +369,28 @@ export class CollectionBrowser
     return this.facetsLoading || this.fullYearAggregationLoading;
   }
 
+  private get mobileFacetsTemplate() {
+    return html`
+      <div id="mobile-filter-collapse">
```

Comment:
idk if we want to play with the `cache` directive here? more of a question to pick your knew knowledge of built-in directives.. ^_^

Reply:
Caching of the HTML would only help if creation was coupled to blocking process, like network or DB. Since we have all the data in browser at create time, no need.